docs/platform/what-is-tizen/overview.md

```diff
@@ -30,22 +30,25 @@ From Tizen 3.0, all profiles are built on top of a common, shared infrastructure
 
 ![Common infrastructure for various profiles](media/about_tizen_2.png)
 
-## IoT Capability
+## Multi Device Experience
```

Comment:
Please use sentence style capitalization for second headings. Ref: https://github.com/Samsung/tizen-docs/blob/master/styleguide/style.md#capitalization
```suggestion
## Multi device experience
```